doc/conf.py

```diff
@@ -180,7 +198,7 @@
         ('Start', 'index'),
         ('Releases', 'releases'),
         ('Installation', 'installation'),
-        ('Manual', 'manual'),
+        #('Manual', 'manual'),
```

Comment:
Wait, why would we no longer render the manual?

Reply:
@KEggensperger , done in #1309, [diff](https://github.com/automl/auto-sklearn/pull/1309/files#diff-e170e9a7d787c21095c6c11bb25f0f1ff0294a42a46d45ba6fb5ed794e457624R184)